python/tool_base/utils.py

```
@@ -66,7 +66,7 @@ def prefit_from_workspace(file, workspace, params, setPars=None):
         gobs = ws.var(p + "_In")
 
         # For pyROOT NULL test: "pdf != None" != "pdf is not None"
-        if pdf != None and gobs != None:
+        if pdf != None and gobs is not None:
```

Ruff (0.14.2)

Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)